fpo-web/src/app/contact/contact.component.html

```diff
@@ -1,3 +1,91 @@
-<div>
-    Contact!
-</div>
\ No newline at end of file
+<div class="row mt-4 mt-md-5">
+  <div class="col-lg-8" id="feedback">
+
+    <h2 tabindex="-1">Submit Feedback</h2>
+
+    <p>Comments and suggestions may be sent using the form below.</p>
+
+    <div class="alert alert-success alert-dismissible" id="alert-success" role="alert" tabindex="0" *ngIf="sent">
+      <h3 class="h5 my-0">Feedback Sent</h3>
+      Thank you, your feedback has been submitted.
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close" (click)="sent=false">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+
+    <div class="alert alert-warning alert-dismissible" id="alert-required" role="alert" tabindex="0"
+      *ngIf="feedback.invalid">
+      <h3 class="h5 my-0">Required Field(s)</h3>
+      One or more fields are missing from the feedback form.
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close" (click)="feedback.invalid=false">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+
+    <div class="alert alert-danger alert-dismissible" id="alert-error" role="alert" tabindex="0" *ngIf="failed">
+      <h3 class="h5 my-0">Error sending feedback</h3>
+      Your request could not be completed at this time.
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close" (click)="failed=false">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+
+    <div class="row form-group mt-2">
+      <div class="col">
+        <label class="control-label mb-0">Contact Reason</label>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="reason" value="incorrect" id="reason_incorrect" required
+            [(ngModel)]="feedback.reason">
+          <label class="form-check-label" for="reason_incorrect">Report incorrect information</label>
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="reason" value="additional" id="reason_additional" required
+            [(ngModel)]="feedback.reason">
+          <label class="form-check-label" for="reason_additional">Request additional information on BC
+            organizations</label>
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="reason" value="signup" id="reason_signup" required
+            [(ngModel)]="feedback.reason">
+          <label class="form-check-label" for="reason_signup">Find out how my government organization can use this</label>
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="reason" value="developer" id="reason_developer" required
+            [(ngModel)]="feedback.reason">
+          <label class="form-check-label" for="reason_developer">Explore clicky things as a software developer</label>
```

Comment:
Remove the *Explore clicky things as a software developer* item.

Reply:
Done